src/wp-includes/class-walker-comment.php

```diff
@@ -244,6 +252,26 @@ protected function ping( $comment, $depth, $args ) {
 		<?php
 	}
 
+	/**
+	 * Remove links from the pending comment's text if the commenter has not consent to the comment cookie.
+	 *
+	 * @since 5.4.2
```

Comment:
Not sure if we should use `@since 5.4.2` or `@since 5.1.6`. It's technically fixed in 5.4.2 but it sounds weird to use 5.4.2’s `@since` mentions in WordPress 5.1.6 🤔

Reply:
We do this frequently when backporting security fixes.

Reply:
Ok, thanks, I wasn't sure of this.